mongodbatlas/resource_mongodbatlas_global_cluster_config.go

```diff
@@ -282,6 +304,14 @@ func removeManagedNamespaces(ctx context.Context, conn *matlas.Client, remove []
 			Db:             mn["db"].(string),
 			CustomShardKey: mn["custom_shard_key"].(string),
 		}
+
+		if isCustomShardKeyHashed, okCustomShard := mn["is_custom_shard_key_hashed"]; okCustomShard {
+			addManagedNamespace.IsCustomShardKeyHashed = pointy.Bool(isCustomShardKeyHashed.(bool))
+		}
+
+		if isShardKeyUnique, okShard := mn["is_shard_"]; okShard {
```

Comment:
is this being tested? it seems it is incomplete

Reply:
yes, It was incomplete